python/ipyleaflet/pyproject.toml

```
@@ -0,0 +1,46 @@
+[build-system]
+requires = [
+    "hatchling",
+    "jupyterlab==4.*",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "ipyleaflet"
+version = "0.18.1"
+description = "A Jupyter widget for dynamic Leaflet maps"
+readme = "README.md"
```

Would this work? That way we get the proper readme on Pypi

```suggestion
readme = "../../README.md"
```